robotis-op2/robot-window.md

```diff
@@ -0,0 +1,18 @@
+# Robot Window
+
+When you double click on the robot, a new window appears. This window is called
+`Robot Window`, it has several tabs allowing you to perform different things.
+First tabs concern the simulation. They will be described here, the last tab is
+used to interact with the real robot and will therefore be described in next
+sections.
+
+You can at any time restart the `Robot Window` by pressing the revert button of
```

Comment:
In this sentence I don't understand what is the point of restarting the robot window and why someone should need it.

In this case the previous sentence was more understandable:
``In the unlikely case of something going wrong with the Robot Window (freeze, bad behavior, etc.) you can at any time restart it by pressing the revert button of the simulation.``